_conferences/gids-2018.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "GIDS"
```

Comment:
Seems like we should use the full name here.

Reply:
Done